plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstance.java

```diff
@@ -187,36 +194,56 @@ public InstanceProcess createProcess(Command command, String outputChannel) thro
     public InstanceKey saveToSnapshot(String owner) throws MachineException {
         try {
             final String repository = generateRepository();
-            String comment = format("Suspended at %1$ta %1$tb %1$td %1$tT %1$tZ %1$tY", System.currentTimeMillis());
-            if (owner != null) {
-                comment = comment + " by " + owner;
+            final String tag = "latest";
+            if (snapshotUseRegistry) {
+                final String digest = saveInRegistry(owner, repository, tag);
+                return new DockerInstanceKey(repository, tag, registry, digest);
+            } else {
+                createImage(owner, repository, tag);
```

Comment:
I think it is better to name that method in the same way as `saveInRegistry` - `saveLocallly`